_layouts/home.html

```diff
@@ -31,5 +31,22 @@ <h1>{{ content }}</h1>
         </div>
     </main>
     {%- include footer.html -%}
+    <script type="text/javascript">
```

Comment:
Does our CSP allow inline scripts?

Reply:
it does for now, but changed to a script file to be safer.